### PR TITLE
RN-246 include NOTICE links in about screen

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR includes the links for the platform and mobile app NOTICES in the about screen.

**NOTE:** The FormattedText component was refactored so it can include child `FormattedText` in the values passed in the props

#### Ticket Link
https://mattermost.atlassian.net/browse/RN-246

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on: 
* IOS 10.3.2 iPhone 6s
* Android 6.0.1 Samsung J5

#### Screenshots
image

### CODE REVIEW

File: app/screens/about/about.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ export default class About extends PureComponent {
         Linking.openURL('http://about.mattermost.com/');
     };
 
+    handlePlatformNotice = () => {
+        Linking.openURL('https://about.mattermost.com/platform-notice-txt/');
```
Comment (jarredwitt) [inline app/screens/about/about.js]: Does this need to come from the config file?

File: app/screens/about/about.js [hunk 2]
```diff
@@ -34,6 +34,14 @@ export default class About extends PureComponent {
...
+    };
+
+    handleMobileNotice = () => {
+        Linking.openURL('https://about.mattermost.com/mobile-notice-txt/');
```
Comment (jarredwitt) [inline app/screens/about/about.js]: same here.

File: app/screens/about/about.js (hunk 1 above)
Comment (enahum) [inline app/screens/about/about.js]: sure I can add those to the config file

**Comment by enahum** [comment]
@hmhealey @jarredwitt done, now it takes the values from the config file

**Pull request merged**